### PR TITLE
Port over ArrowDateTimeField fix

### PR DESCRIPTION
**Pull request by nnewman**
Bug Fixes:

* Adds fix for string values being passed in to the database

### CODE REVIEW

File: fleaker/peewee/fields/arrow.py [hunk 1]
```diff
@@ -73,15 +73,17 @@ def python_value(self, value):
...
 
         return value
 
     def db_value(self, value):
         """Convert the Arrow instance to a datetime for saving in the db."""
-        if isinstance(value, arrow.Arrow):
+        if isinstance(value, string_types):
+            value = arrow.get(value)
+
+        if isinstance(value, arrow.arrow.Arrow):
```
Comment (eligundry) [inline fleaker/peewee/fields/arrow.py]: There is no need to nest `arrow.arrow.Arrow` when `arrow.Arrow` just works.

File: fleaker/peewee/fields/arrow.py [hunk 2]
```diff
@@ -73,15 +73,17 @@ def python_value(self, value):
         """
         value = super(ArrowDateTimeField, self).python_value(value)
 
-        if (isinstance(value, (datetime.datetime, datetime.date,
-                               string_types))):
-            value = arrow.get(value)
+        if isinstance(value, (datetime.datetime, datetime.date, string_types)):
```
Comment (eligundry) [inline fleaker/peewee/fields/arrow.py]: Why did this change? It exceeds 80 chars.

File: fleaker/peewee/fields/arrow.py (hunk 1 above)
Comment (nnewman) [inline fleaker/peewee/fields/arrow.py]: Fixed in 4de5ac9

File: fleaker/peewee/fields/arrow.py (hunk 2 above)
Comment (nnewman) [inline fleaker/peewee/fields/arrow.py]: According to intellij it is exactly 80 characters as is?

Comment (eligundry) [inline fleaker/peewee/fields/arrow.py]: It's at 81 according to pylint.

Comment (nnewman) [inline fleaker/peewee/fields/arrow.py]: Fixing. That's so weird. I don't believe you pylint.

2859205

**Pull request merged**